docs/book/src/user/quick-start.md

```diff
@@ -854,6 +859,7 @@ See the [clusterctl] documentation for more detail about clusterctl supported ac
 [management cluster]: ../reference/glossary.md#management-cluster
 [Metal3 getting started guide]: https://github.com/metal3-io/cluster-api-provider-metal3/blob/master/docs/getting-started.md
 [Metal3 provider]: https://github.com/metal3-io/cluster-api-provider-metal3/
+[oci-provider]: https://github.com/oracle/cluster-api-provider-oci
```

Comment:
Up to you, but you might want to link to the book instead: https://oracle.github.io/cluster-api-provider-oci/#getting-started

Reply:
Good call. Fixed.